drivers/interrupt_controller/exti_stm32.c

```diff
@@ -51,7 +51,51 @@
 #define EXTI_LINES 49
 #endif
 
-#if defined(CONFIG_SOC_SERIES_STM32MP1X)
+#if defined(CONFIG_SOC_SERIES_STM32F0X) || \
+    defined(CONFIG_SOC_SERIES_STM32L0X)
+IRQn_Type exti_irq_table[] = {
```

Comment:
Adding const would save some RAM, no? Here and elsewhere.

Reply:
Sure, let me have a try. Thanks!